amundsen_application/static/js/utils/tableissues-utils.ts

```diff
@@ -0,0 +1,10 @@
+import { ASSOCIATION_TEXT, MAX_TEXT_LENGTH } from '../components/TableDetail/TableIssues/constants';
+
+export function truncateText(issueTitle: string) : string {
+  if (!issueTitle) {
+    return issueTitle; 
+  }
+  const truncated = issueTitle.length > MAX_TEXT_LENGTH ? 
+    issueTitle.substring(0, MAX_TEXT_LENGTH)  + "...": issueTitle;  
+  return '"' + truncated + '"' +  ASSOCIATION_TEXT; 
```

Comment:
Instead of truncating the text displayed, we can use the `truncated` css class to add the `...` where it's needed. This will also automatically show/hide more characters if the width changes.

You can check the `TableListItem` for an example. (For some reason we double wrap the `truncated` text, but I'm not 100% sure that's necessary.)

Reply:
good point, i hadn't done it initially because of the additional text required in the mocks. but i'll change it to use ellipsis instead